config/initializers/dynflow.rb

```diff
@@ -1,2 +1,15 @@
 # Ignore Dynflow tables when schema-dumping. Dynflow tables are handled automatically by Dynflow.
-ActiveRecord::SchemaDumper.ignore_tables << '^dynflow_*'
+# Ruby dumper recognizes table names and regular expression (as string), PostgreSQL dumper only
+# recognizes table names as it passes this to pg_dump via arguments.
+#
+ActiveRecord::SchemaDumper.ignore_tables = [
+  '^dynflow_.*$',
```

Comment:
Is this necessary now that we're listing all the tables explicitly?

Reply:
@lzap ping

Reply:
No I guess, wanted to be future proof because we will forget to update it. But sure let's be explicit.